graas/hardware.py

```diff
@@ -0,0 +1,85 @@
+from twisted.internet import task, reactor
+from twisted.internet.defer import succeed, inlineCallbacks, returnValue
+
+
+class AsyncIO(object):
+
+    def __init__(self, GPIO):
+        '''
+        GPIO - Raspberry Pi GPIO module to control hardware
+        '''
+        GPIO.setmode(GPIO.BOARD)
+        self.pin_state = {}
+
+        self.INPUT = GPIO.IN
+        self.OUTPUT = GPIO.OUT
+        self.HIGH = GPIO.HIGH
+        self.LOW = GPIO.LOW
+        self.PULL_UP = GPIO.PUD_UP
+        self.PULL_DOWN = GPIO.PUD_DOWN
```

Comment:
If we put these constants on the class, we can use them as default values in function definitions directly, e.g.
```
def input(self, pin, pull=PULL_DOWN):
    ...
```